### PR TITLE
Project Overview with react-table v8

### PR DESCRIPTION
**Pull request by Tymek**

### CODE REVIEW

File: frontend/src/component/common/Table/cells/DateCell/DateCell.tsx [hunk 1]
```diff
@@ -5,11 +5,13 @@ import { getLocalizedDateString } from '../../../util';
 
 interface IDateCellProps {
     value?: Date | string | null;
+    getValue?: () => Date | string | null | undefined;
 }
 
-export const DateCell: VFC<IDateCellProps> = ({ value }) => {
+export const DateCell: VFC<IDateCellProps> = ({ value, getValue }) => {
```
Comment (Tymek) [inline frontend/src/component/common/Table/cells/DateCell/DateCell.tsx:11]:
```suggestion
// `value` is for react-table (v7) and `getValue is for @tanstack/react-table (v8)
export const DateCell: VFC<IDateCellProps> = ({ value, getValue }) => {
```

File: frontend/src/component/common/Table/cells/FeatureNameCell/FeatureNameCell.tsx [hunk 2]
```diff
@@ -4,17 +4,17 @@ import { LinkCell } from 'component/common/Table/cells/LinkCell/LinkCell';
...
     };
 }
 
 export const FeatureNameCell: VFC<IFeatureNameCellProps> = ({ row }) => (
     <LinkCell
-        title={row.original.name}
-        subtitle={row.original.description}
+        title={row.original.name || ''}
+        subtitle={row.original.description || ''}
         to={`/projects/${row.original.project}/features/${row.original.name}`}
```
Comment (Tymek) [inline frontend/src/component/common/Table/cells/FeatureNameCell/FeatureNameCell.tsx:18]: Wrap in "if" - don't render link if it is incorrect

File: frontend/src/component/common/Table/cells/FeatureSeenCell/FeatureEnvironmentSeenCell.tsx [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import React, { VFC } from 'react';
 import { FeatureEnvironmentSeen } from 'component/feature/FeatureView/FeatureEnvironmentSeen/FeatureEnvironmentSeen';
-import { FeatureSchema } from 'openapi';
+import { FeatureSchema, FeatureSearchResponseSchema } from 'openapi';
 
 interface IFeatureSeenCellProps {
-    feature: FeatureSchema;
+    feature: FeatureSchema | FeatureSearchResponseSchema;
```
Comment (Tymek) [inline frontend/src/component/common/Table/cells/FeatureSeenCell/FeatureEnvironmentSeenCell.tsx:6]: "Pick" what is needed

File: frontend/src/component/common/Table/cells/FeatureTypeCell/FeatureTypeCell.tsx [hunk 4]
```diff
@@ -15,15 +16,17 @@ const StyledContainer = styled('div')(({ theme }) => ({
     color: theme.palette.text.disabled,
 }));
 
-export const FeatureTypeCell: VFC<IFeatureTypeProps> = ({ value }) => {
+export const FeatureTypeCell: VFC<IFeatureTypeProps> = ({
```
Comment (Tymek) [inline frontend/src/component/common/Table/cells/FeatureTypeCell/FeatureTypeCell.tsx:19]: Backwards compatibility comment

**Review by kwasniew — APPROVED** [review]
Pair reviewed

**Pull request merged**